JanePhotoBrowser/PhotoBrowser/PhotoBrowserFullscreen/PhotoBrowserStyleKit.swift

```diff
@@ -9,6 +9,11 @@
 import UIKit
 
 public class PhotoBrowserIconography: NSObject {
+
+    @available(iOS 13, *)
+    public static func imageOfXIcon() -> UIImage? {
+        return UIImage(systemName: "xmark")
```

Comment:
When in a pod you must specify the bundle or it will not work once you try to integrate with the app